### PR TITLE
docs(README): add OpenCollective badge and section

### PR DESCRIPTION
**Pull request by ylemkimon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
...
 Learn more about using KaTeX [on the website](https://katex.org)!
 
-## Contributing
+## Contributors
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
```
Comment (edemaine) [inline README.md:92]: Is this plot supposed to do something? I don't see anything (at least in GitHub dark mode).

File: README.md [hunk 2]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
 
 Learn more about using KaTeX [on the website](https://katex.org)!
 
-## Contributing
+## Contributors
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```
Comment (edemaine) [inline README.md:91]: I think this sentence needs revising now that there are many senses of contribution.  How about something like:

```suggestion
This project exists thanks to all the people who contribute code. If you'd like to help, see [our guide to contributing code](CONTRIBUTING.md).
```

File: README.md [hunk 3]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
...
+<a href="https://opencollective.com/katex/organization/2/website"><img src="https://opencollective.com/katex/organization/2/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/3/website"><img src="https://opencollective.com/katex/organization/3/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/4/website"><img src="https://opencollective.com/katex/organization/4/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/5/website"><img src="https://opencollective.com/katex/organization/5/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/6/website"><img src="https://opencollective.com/katex/organization/6/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/7/website"><img src="https://opencollective.com/katex/organization/7/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/8/website"><img src="https://opencollective.com/katex/organization/8/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/9/website"><img src="https://opencollective.com/katex/organization/9/avatar.svg"></a>
```
Comment (edemaine) [inline README.md:115]: These take up a lot of space, but I guess it'll look nicer once we have contributors.

**Review by edemaine — COMMENTED** [review]
Nice idea to link to OpenCollective in the README as well! Some comments and questions.

File: README.md (hunk 1 above)
Comment (ylemkimon) [reply README.md:92]: It should show contributors like this, but due to https://github.com/opencollective/opencollective/issues/4517 it doesn't work now:

File: README.md [hunk 4]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
...
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
+
+### Financial Contributors
+
+Become a financial contributor and help us sustain our community. [[Contribute](https://opencollective.com/katex/contribute)]
```
Comment (ylemkimon) [inline README.md:96]:
```suggestion
Become a financial contributor and help us sustain our community.
```

File: README.md [hunk 5]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
...
+
+#### Individuals
+
+<a href="https://opencollective.com/katex"><img src="https://opencollective.com/katex/individuals.svg?width=890"></a>
+
+#### Organizations
+
+Support this project with your organization. Your logo will show up here with a link to your website. [[Contribute](https://opencollective.com/katex/contribute)]
```
Comment (ylemkimon) [inline README.md:104]:
```suggestion
Support this project with your organization. Your logo will show up here with a link to your website.
```

File: README.md [hunk 6]
```diff
@@ -88,20 +88,20 @@ Learn more about using KaTeX [on the website](https://katex.org)!
 
 ### Code Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+This project exists thanks to all the people who contribute code. If you'd like to help, see [our guide to contributing code](CONTRIBUTING.md).
 <a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
 
 ### Financial Contributors
 
-Become a financial contributor and help us sustain our community. [[Contribute](https://opencollective.com/katex/contribute)]
```
Comment (edemaine) [inline README.md]: Maybe add the link to "Become a financial contributor"?

File: README.md (hunk 1 above)
Comment (edemaine) [reply README.md:92]: Reading some new posts on that thread, it looks like https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false works:

Comment (ylemkimon) [reply README.md:92]:
```suggestion
<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
```

File: README.md (hunk 6 above)
Comment (ylemkimon) [reply README.md]: The last image will show as a button.

Comment (edemaine) [reply README.md]: For accessibility, we should add an `alt` attribute to said button, or add the link to the text.

File: README.md [hunk 7]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
...
+This project exists thanks to all the people who contribute code. If you'd like to help, see [our guide to contributing code](CONTRIBUTING.md).
+<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
+
+### Financial Contributors
+
+Become a financial contributor and help us sustain our community.
+
+#### Individuals
+
+<a href="https://opencollective.com/katex"><img src="https://opencollective.com/katex/individuals.svg?width=890"></a>
```
Comment (ylemkimon) [inline README.md:100]:
```suggestion
<a href="https://opencollective.com/katex"><img src="https://opencollective.com/katex/individuals.svg?width=890" alt="Contribute on Open Collective"></a>
```

File: README.md [hunk 8]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
...
+<a href="https://opencollective.com/katex/organization/2/website"><img src="https://opencollective.com/katex/organization/2/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/3/website"><img src="https://opencollective.com/katex/organization/3/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/4/website"><img src="https://opencollective.com/katex/organization/4/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/5/website"><img src="https://opencollective.com/katex/organization/5/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/6/website"><img src="https://opencollective.com/katex/organization/6/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/7/website"><img src="https://opencollective.com/katex/organization/7/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/8/website"><img src="https://opencollective.com/katex/organization/8/avatar.svg"></a>
+<a href="https://opencollective.com/katex/organization/9/website"><img src="https://opencollective.com/katex/organization/9/avatar.svg"></a>
```
Comment (ylemkimon) [inline README.md:115]:
```suggestion
<a href="https://opencollective.com/katex/organization/0/website"><img src="https://opencollective.com/katex/organization/0/avatar.svg" alt="Organization 1"></a>
<a href="https://opencollective.com/katex/organization/1/website"><img src="https://opencollective.com/katex/organization/1/avatar.svg" alt="Organization 2"></a>
<a href="https://opencollective.com/katex/organization/2/website"><img src="https://opencollective.com/katex/organization/2/avatar.svg" alt="Organization 3"></a>
<a href="https://opencollective.com/katex/organization/3/website"><img src="https://opencollective.com/katex/organization/3/avatar.svg" alt="Organization 4"></a>
<a href="https://opencollective.com/katex/organization/4/website"><img src="https://opencollective.com/katex/organization/4/avatar.svg" alt="Organization 5"></a>
<a href="https://opencollective.com/katex/organization/5/website"><img src="https://opencollective.com/katex/organization/5/avatar.svg" alt="Organization 6"></a>
<a href="https://opencollective.com/katex/organization/6/website"><img src="https://opencollective.com/katex/organization/6/avatar.svg" alt="Organization 7"></a>
<a href="https://opencollective.com/katex/organization/7/website"><img src="https://opencollective.com/katex/organization/7/avatar.svg" alt="Organization 8"></a>
<a href="https://opencollective.com/katex/organization/8/website"><img src="https://opencollective.com/katex/organization/8/avatar.svg" alt="Organization 9"></a>
<a href="https://opencollective.com/katex/organization/9/website"><img src="https://opencollective.com/katex/organization/9/avatar.svg" alt="Organization 10"></a>
```

File: README.md [hunk 9]
```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
...
 Learn more about using KaTeX [on the website](https://katex.org)!
 
-## Contributing
+## Contributors
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+### Code Contributors
+
+This project exists thanks to all the people who contribute code. If you'd like to help, see [our guide to contributing code](CONTRIBUTING.md).
+<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
```
Comment (ylemkimon) [inline README.md:92]:
```suggestion
<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false" alt="Code contributors" /></a>
```

**Review by edemaine — APPROVED** [review]
Thanks!

**Pull request merged**